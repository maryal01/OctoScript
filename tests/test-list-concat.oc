list<string> l1 = ["hari", "krishna", "ramayan"];
list<string> l2 = ["rooney", "tevez", "beckham", "adebayor"];
list<string> l3 = l1.concat(l2);
int i = 0;
int length = l3.length();
while(i < length){
    print(l3.get(i));
    print("\n");
    i = i + 1;
}