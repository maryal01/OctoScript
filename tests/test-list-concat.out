hari
krishna
ramayan
rooney
tevez
beckham
adebayor
